judgments/views/advanced_search.py

```diff
@@ -163,8 +163,14 @@ def advanced_search(request: HttpRequest) -> HttpResponse:
                 ) = process_court_facets(search_response.facets, courts_and_tribunals)
                 unprocessed_facets, year_facets = process_year_facets(unprocessed_facets)
 
-            changed_queries = {key: value for key, value in params.items() if value is not None and not key == "page"}
+            changed_queries = {}
+            for key in params:
+                if params.getlist(key) and key != "page":
```

Comment:
I'd consider setting params.getlist(key) to a variable, but this is splitting hairs.

Reply:
The typing here feels weird -- either things are multielement lists or not a list at all.

Reply:
Good point - have simplified this